aws/service.go

```diff
@@ -1749,10 +1749,32 @@ func WorkspacesService(ctx context.Context, d *plugin.QueryData) (*workspaces.Wo
 }
 
 func getSession(ctx context.Context, d *plugin.QueryData, region string) (*session.Session, error) {
-	return getSessionWithMaxRetries(ctx, d, region, 9)
+	awsConfig := GetConfig(d.Connection)
+
+	// As per the logic used in retryRules of NewConnectionErrRetryer with minimum delay of 25ms and maximum
+	// number of retries as 9 (our default), the default maximum delay will not be more than approximately 3 minutes to avoid steampipe
+	// waiting too long to render result
+	maxRetries := 9
+	var minErrRetryDelay time.Duration = 25 * time.Millisecond // Default minimum delay
+
+	// Set max retry count from configuration file
```

Comment:
Can you please add support for the `AWS_MAX_ATTEMPTS` env variable (as detailed in https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-retries.html#cli-usage-retries-configure) here? The AWS Go SDK v1 doesn't support it, so we should check it here using the follow order of precedence:
- Check config arg, if set use that; else
- Check env variable, if set use that; else
- Fall back to default (9 attempts)